build-aux/flatpak/com.github.akiraux.akira.json

```diff
@@ -23,7 +23,16 @@
     "--filesystem=xdg-run/gvfs",
     "--filesystem=xdg-run/gvfsd"
   ],
-  "modules": [{
+  "modules": [
+   {
+      "name": "libportal",
+      "buildsystem": "meson",
+      "sources": [{
+        "type": "git",
+        "url": "https://github.com/flatpak/libportal.git"
```

Comment:
should ideally be a tarball of a tagged release, at worst a specific commit

Reply:
Ok.